.github/workflows/check.yml

```diff
@@ -181,13 +181,13 @@ jobs:
       fail-fast: false
       matrix:
         os:
-          - macos-latest
-          - ubuntu-latest
-          - windows-latest
+          - macos-11
+          - ubuntu-20.04
+          - windows-2022
     steps:
       - name: Harden runner
         uses: step-security/harden-runner@ebacdc22ef6c2cfb85ee5ded8f2e640f4c776dd5 # v2.0.0
-        if: ${{ matrix.os == 'ubuntu-latest' }}
+        if: ${{ matrix.os == 'ubuntu-20.04' }}
```

Comment:
Ideally this doesn't need to be changed when the matrix is updated. Either this step can just be run for any OS (provided it doesn't error) or it may make more sense to check if `matrix.os` starts with `'ubuntu-'`.